shared/js/otSdkDriver.js

```diff
@@ -622,6 +624,7 @@ loop.OTSdkDriver = (function() {
         srcMediaElement: sdkSubscriberVideo
       }));
 
+      this._notifyMetricsEvent("Session.subscribeCompleted");
```

Comment:
It was mainly so it was logged near the media connected that happens. It doesn't really matter much though anyway. The bits in the middle are unlikely to blow up unless there's a programming error which would be found before shipping (dispatcher calls are protected).